eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -73,14 +73,20 @@ public function load($id, $version, array $translations = null);
      * Missing items (NotFound) will be missing from the array and not cause an exception, it's up
      * to calling logic to determine if this should cause exception or not.
      *
-     * NOTE: Even if LoadStruct technically allows to load several versions of same content, this method does not allow
-     * this by design as content is returned as Map with key being content id.
+     * If items are missing but for other reasons then not being found, for instance exceptions during loading field
+     * data. Then the exception will be logged as warning or error depending on severity
+     * (TODO: Andrew should expand on the cases here, if it's just a matter of missing fieldtypes then maybe we should throw? In case of missing binary files, how was this handled in IO?)
```

Comment:
:grinning: 
Ok. So based on my experience with Customer's data, I would add here:
```
     * The most common case of possible exceptions during loading of Content data is migration,
     * where either custom Field Type configuration or implementation might not be aligned with new
     * version of the system.
```
It rather has to be that general, because it's hard to guess what exactly Developer did in his custom Field Type implementation (my guess is that it will apply to custom Field Type Converter, but might be also external storage GW implementation itself).

Note that there might be just a few of erroneous Content items which wouldn't load in any case, so I really don't think we should throw there anything affecting entire batch.